src/redux/categories/categories.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 
-const Categories = () => (
-  <div className="cat-list">
-    <p>Under Construction</p>
-  </div>
-);
+function Categories() {
+  return (
+    <div className="cat-list">
+      <p>Under Construction</p>
+    </div>
+  );
```

Comment:
- [ ] please verify all the files because sometimes you use the arrow function and some files you don"t.

Reply:
Hello @dasileker,
Thank you for the feedback. I have an issue when validating props, I keep on getting an error on the console saying the prop is validated with "isRequired" but the value is undefined, and it crashes my application. Do you please know how I could walk around that, or any additional resources that will help me out? 

Thank you